stock_reserve/view/stock_reserve.xml

```diff
@@ -69,8 +69,8 @@
                             <field name="location_id" />
                             <field name="location_dest_id" />
                         </group>
-                        <group name="note" string="Notes">
-                            <field name="note" nolabel="1" />
+                        <group name="note" colspan="2">
```

Comment:
I agree for the removing of `nolabel=1`

why removing the `string="Notes"` group name ?

Reply:
It's superfluous as the field is also called notes. So it would show notes twice.